### PR TITLE
chore(news): create hexo-fs 1.0.1 & hexo-renderer-less 1.0.0 released…

### PR DESCRIPTION
**Pull request by YoshinoriN**
I create official hexo release news.
Could you please someone review it ?

And how do you think about create official plugins release news ?
If good, I will create news about official plugin every release.

### CODE REVIEW

**Comment by tcrowe** [comment]
Yeah, I would like to see these updates and I think others might be interested to see changes and progress on the blog.

**Comment by tomap** [comment]
I think it's great to publish more on the blog. Thank you.

File: source/_posts/2018-12-01-2018-12-01-hexofs-1-0-1-and-hexo-renderer-less-1-0-0-released-md.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Official plugin hexo-fs 1.0.1 & hexo-renderer-less 1.0.0 Released!!
+---
+
+We released official plugins hexo-fs & hexo-renderer-less new version.
```
[the remark this answers is not in the document]
Comment (YoshinoriN) [reply source/_posts/2018-12-01-2018-12-01-hexofs-1-0-1-and-hexo-renderer-less-1-0-0-released-md.md]: Thanks !

**Comment by YoshinoriN** [comment]
@tcrowe @ertrzyiks @tomap 
Merged ! Thank you very much !
I will continue to create release news of official plugin.

**Comment by tomap** [comment]
@YoshinoriN super!

**Pull request merged**